kube_deploy/Stag/deploy.yml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dcs-app-stag
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: dcs-app-stag
+    spec:
+      containers:
+        - name: data-capture-service
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cica/cica-repo-stag:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3100
+          env:
+            - name: DCS_JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: data-capture-service-secrets
+                  key: dcs_jwt_secret
+            - name: DATABASE_URL_WITH_CREDENTIALS
+              valueFrom:
+                secretKeyRef:
+                  name: shared-secrets
+                  key: database_url_with_credentials
+            - name: DCS_LOG_LEVEL
+              value: info
+            - name: DCS_SQS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: dcs-sqs-secret
+                  key: access_key_id
+            - name: DCS_SQS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: dcs-sqs-secret
+                  key: secret_access_key
+            - name: AWS_SQS_ARN
+              valueFrom:
+                secretKeyRef:
+                  name: application-sqs
+                  key: sqs_arn
+            - name: AWS_SQS_ID
+              valueFrom:
+                secretKeyRef:
+                  name: application-sqs
+                  key: sqs_id
+            - name: AWS_SQS_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: application-sqs
+                  key: sqs_name
+            - name: NOTIFY_AWS_SQS_ID
+              valueFrom:
+                secretKeyRef:
+                  name: notify-sqs
+                  key: sqs_id
+            - name: MESSAGE_BUS_URL
+              valueFrom:
+                secretKeyRef:
+                  name: shared-secrets
+                  key: message_bus_url
+            - name: MESSAGE_BUS_CREDENTIALS
+              valueFrom:
+                secretKeyRef:
+                  name: shared-secrets
+                  key: message_bus_credentials
+            - name: NOTIFY_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: shared-secrets
+                  key: notify_api_key
+            - name: DCS_URL
+              value: https://data-capture-service.stag.claim-criminal-injuries-compensation.service.justice.gov.uk
+            - name: APP_ENV
+              value: stag
+            - name: DCS_SESSION_DURATION
```

Comment:
It looks like the `DCS_SESSION_DURATION` has been [updated in the other configs](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/blob/da754e87bf94ca128961ae092e1cac958db45e79/kube_deploy/Prod/deploy.yml#L80) since the PR was raised. Unless the `1800000` value is specifically required for staging, can this be updated to reflect the other configs?

Reply:
Updated!

Reply:
👍